packages/svelte/src/internal/client/reactivity/types.d.ts

```diff
@@ -40,7 +40,7 @@ export interface Effect extends Reaction {
 	/** The associated component context */
 	ctx: null | ComponentContext;
 	/** The effect function */
-	fn: () => void | (() => void);
+	fn: null | (() => void) | (() => void);
```

Comment:
```suggestion
	fn: null | (() => void | (() => void));
```
It looks like `fn` previously returned void or a function returning void.
That seemed to get changed to return void or to return void. I have the feeling a bracket is off

Reply:
good catch, thanks!